### PR TITLE
Js suite level test change

### PR DESCRIPTION
**Pull request by jsingle**
@paetling move `compose` and `services` to suite level.  Still working on dusty-specs PR

### CODE REVIEW

File: dusty/commands/test.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def _services_compose_up(expanded_specs, app_or_lib_name, testing_spec, suite_na
         previous_container_names.append("{}_{}_1".format(_compose_project_name(app_or_lib_name, suite_name), service_name))
     return previous_container_names
 
-def _app_or_lib_compose_up(testing_spec, app_or_lib_name, app_or_lib_volumes, test_command, previous_container_name, suite_name):
+def _app_or_lib_compose_up(base_compose_spec, app_or_lib_name, app_or_lib_volumes, test_command, previous_container_name, suite_name):
```
Comment (paetling) [inline dusty/commands/test.py]: testing_base_compose_spec?  I thought this was at the level of gcweb, not gcweb.tests.  This could just be me though

**Comment by paetling** [comment]
fix tests.  But this looks pretty good

**Comment by thieman** [comment]
LGTM

File: dusty/commands/test.py (hunk 1 above)
Comment (jsingle) [inline dusty/commands/test.py]: I'll rename it.  Not entirely sure what you mean, but this is the compose dict from gcweb.tests.suite.compose (for example)

**Pull request merged**